.github/workflows/workflow-markdown-check.md

```diff
@@ -1,4 +1,6 @@
 ## Reusable Workflow for Standardization Validation
 
    - **Purpose:** Validate the organization's standards across projects.
-   - **Usage:** Call this workflow while making sure the necessary secrets are inherited for successful execution.
\ No newline at end of file
+   - **Steps**
```

Comment:
the workflow has one optional input config-file-path; please document usage.

Reply:
done